src/iomem.rs

```diff
@@ -0,0 +1,60 @@
+#[cfg(feature = "serde1")]
+use serde::{Deserialize, Serialize};
+use std::io;
+
+use super::{FileWrapper, ProcResult};
+use crate::split_into_num;
+
+pub struct IoMem;
```

Comment:
It seems a little weird to have an empty struct here.  Instead of `IoMem::new()` what about `fn iomem() -> ProcResult<Vec<PhysicalMemoryMap>>`?  This would be consistent with other functions that return a list of things (like [diskstats()](https://docs.rs/procfs/0.14.1/procfs/fn.diskstats.html) and [locks()](https://docs.rs/procfs/0.14.1/procfs/fn.locks.html))